assets/src/components/detours/detourRouteSelectionPanel.tsx

```diff
@@ -26,6 +26,10 @@ type SelectedRouteInfo =
 interface DetourRouteSelectionPanelProps {
   allRoutes: Route[]
   selectedRouteInfo: SelectedRouteInfo
+
+  onConfirm?: () => void
```

Comment:
Silly question - do these need to be optional? Seems like in the real world, all of these functions would need to be there.

Reply:
If they're required, we'll have to add simple noop functions to the storybook stories for this component, but that might still be a good idea.

Reply:
I was going to merge without this and open another PR, but I didn't realize that this component wasn't used anywhere else, so I've added it in [`bd50e11` (#2631)](https://github.com/mbta/skate/pull/2631/commits/bd50e111d3eab5315b3000a6df2eae6a80a038c0)